std-bits/base/src/main/java/org/enso/base/enso_cloud/EnsoKeyValuePair.java

```diff
@@ -0,0 +1,15 @@
+package org.enso.base.enso_cloud;
+
+public record EnsoKeyValuePair(String key, String value, String secretId) {
+  public static EnsoKeyValuePair ofNothing(String key) {
+    return new EnsoKeyValuePair(key, null, null);
+  }
+
+  public static EnsoKeyValuePair ofText(String key, String value) {
+    return new EnsoKeyValuePair(key, value, null);
+  }
+
+  public static EnsoKeyValuePair ofSecret(String key, String secretId) {
+    return new EnsoKeyValuePair(key, null, secretId);
+  }
+}
```

Comment:
What happens if I store both a secretId and a value?

Shall we disallow this in the constructor?

I'm not sure if it would not be clearer to split this into 2 classes:
```
public sealed interface EnsoKeyValuePair permits EnsoKeySecretPair, EnsoKeyStringPair {
  static EnsoKeyValuePair ofNothing(String key) {
    return new EnsoKeyStringPair(key, null);
  }

  static EnsoKeyValuePair ofText(String key, String value) {
    return new EnsoKeyStringPair(key, value);
  }

  static EnsoKeyValuePair ofSecret(String key, String secretId) {
    return new EnsoKeySecretPair(key, secretId);
  }
}
public class EnsoKeySecretPair implements EnsoKeyValuePair {
   private final String key;
   private final String secretId;
   ...
}
public class EnsoKeyStringPair implements EnsoKeyValuePair {
   private final String key;
   private final String value;
   ...
}
```